webserver/templates/datasets/eval-job.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+
+{% block title %}Dataset "{{ dataset['name'] }}" by {{ author.musicbrainz_id }} - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <div id="evaluation-results">
+
+    {% from 'datasets/macros.html' import print_header %}
+    {{ print_header(dataset, author, current_user, 'evaluation') }}
+
+    {% if job['status'] == 'pending' %}
+      <div id="status" class="alert alert-info">
+        This dataset has not been evaluated yet.
+      </div>
+
+    {% elif job['status'] == 'running' %}
+      <div id="status" class="alert alert-info">
+        This dataset is being evaluated right now. Results should be available soon.
+      </div>
+
+    {% elif job['status'] == 'failed' %}
+      <div id="status" class="alert alert-danger">
+        Evaluation of this dataset has failed!
```

Comment:
A stack trace is never user-friendly ;)  Ideally we should understand all of the errors, and be able to describe them nicely (with a suggested fix). We can always send the stack trace to us so that we can debug it further.

It might be nice to have an admin interface which lets us see, at least, all pending, running, and finished jobs, with links to them, and the ability to change the state. We'll need admin users to be able to do this too. Make an issue for it, but make it quite low priority. For now we need an easy way to re-queue a stuck process. Maybe we could just have a "resubmit" button for people to do it themselves, but in the future we remove it. What do you think?